keras/losses.py

```diff
@@ -1496,6 +1500,10 @@ def mean_absolute_percentage_error(y_true, y_pred):
   diff = tf.abs(
       (y_true - y_pred) / backend.maximum(tf.abs(y_true),
                                           backend.epsilon()))
+  if ignore_zeros is True:
+    mask = y_true != 0
+    diff = diff[mask]
```

Comment:
Is this intended? It will fill some 0s to diff, and those zeros are still taking into account when it does the mean. From the docstring above, I thought you are going to "ignore" the zeros.